core/scss/utilities/mixins/grid/_centered-column.scss

```diff
@@ -10,11 +10,17 @@
 @mixin centered-column($grid: $grid-media, $screens: $grid-screens, $gutters: $grid-gutters) {
   @include margin(0 auto);
   @each $code, $breakpoint in $grid {
-    $max-width: map-get($screens, $code);
     $gutter: map-get($gutters, $code);
 
     @media #{$breakpoint} {
-      max-width: $max-width - $gutter;
+      @if breakpoint-max($code, $grid) == null {
+        $max-width: map-get($screens, $code);
+        max-width: $max-width - $gutter;
+        width: $max-width - $gutter;
+      } @else {
+        max-width: calc(100% - #{$gutter});
```

Comment:
Actually I believe the side "margin" values are different from the gutter values. These are the numbers I got from Figma (under Visual Guidelines -> Responsive Notes). John confirmed the numbers are for margins on each side so total subtraction would be below numbers x 2:
XS = 20px
SM = 30px
MD = 50px
LG = 80px
XL = 100px
XXL = max 1500px (doesn't use fluid margin since max-width is fixed to 1500px)

Reply:
![screen shot 2019-01-23 at 3 09 19 pm](https://user-images.githubusercontent.com/8933670/51643316-e309de00-1f20-11e9-969a-7ae188d9e7ae.png)

Reply:
Ok, I have adjusted the modular margin variable to match the margins provided here and updated the mixin to reflect that. Please have another review.